protocol/packages/currencies/src/test.rs

```diff
@@ -7,7 +7,7 @@ use crate::lease::astroport::{Atom as LC1, Dydx as LC4, Ntrn as LC2, StAtom as L
     feature = "osmosis-osmosis-usdc_axelar",
     feature = "osmosis-osmosis-usdc_noble"
 ))]
-use crate::lease::osmosis::{Atom as LC1, Axl as LC2, Cro as LC3, Osmo as LC4, Weth as LC5};
+use crate::lease::r#impl::{Atom as LC1, Axl as LC2, Cro as LC3, Osmo as LC4, Weth as LC5};
```

Comment:
As discussed before, I'm opposed to using those currencies instead of a new set, for which if the groups are the problem, we can just have a `#[cfg(feature = "testing")]` to decide if we want the real ones or not.  
My reasoning is that we have absolutely no guarantees what currencies there will be, especially when generating them.

Reply:
the deficiency of this approach is that the production code of groups would be polluted with feature-gated sections enumerating two different sets - the production, and the test one

Reply:
I'd rather have three or four `#[cfg(any(test, feature = "testing"))]`s than having to "fix" the CI to include special cases when we need a build.

Reply:
addressed in move-predefined-protocols-aside branch